packages/main/src/TimePickerBase.ts

```diff
@@ -243,11 +301,81 @@ class TimePickerBase extends UI5Element {
 		this._isPickerOpen = false;
 	}
 
-	async _handleInputClick() {
+	async onResponsivePopoverAfterOpen() {
+		this._isPickerOpen = true;
+		const responsivePopover = await this._getPopover();
+		responsivePopover.querySelector<TimeSelectionClocks>("[ui5-time-selection-clocks]")!._focusFirstButton();
+	}
+
+	/**
+	 * Opens the Inputs popover.
+	 * @async
+	 * @public
+	 * @method
+	 * @name sap.ui.webc.main.TimePickerBase#openInputsPopover
+	 * @returns {Promise} Resolves when the picker is open
+	 */
+	async openInputsPopover() {
+		this.tempValue = this.value && this.isValid(this.value) ? this.value : this.getFormat().format(new Date());
+		const popover = await this._getInputsPopover();
+		popover.showAt(this);
+		this._isInputsPopoverOpen = true;
+	}
+
+	/**
+	 * Closes the Inputs popover
+	 * @public
+	 * @method
+	 * @name sap.ui.webc.main.TimePickerBase#closeInputsPopover
+	 */
+	async closeInputsPopover() {
+		const popover = await this._getInputsPopover();
+		popover.close();
+	}
+
+	toggleInputsPopover() {
+		if (this.isInputsPopoverOpen()) {
+			this.closeInputsPopover();
+		} else if (this._canOpenInputsPopover()) {
+			this.openInputsPopover();
+		}
+	}
+
+	/**
+	 * Checks if the inputs popover is open
+	 * @public
+	 * @method
+	 * @name sap.ui.webc.main.TimePickerBase#isInputsPopoverOpen
+	 * @returns {boolean}
+	 */
+	isInputsPopoverOpen() {
```

Comment:
do we need this method as public?

Reply:
There is the same method for other popover (with Clocks), so I assume if it is public, we should have the same method for Inputs popover.